manifests/init.pp

```diff
@@ -119,7 +119,10 @@
       icvpn_as => $mesh_as;
     }
   }
- 
+  
+  ffnord::resources::ffnord::field {
+    "MESH_CODE": value => "${mesh_code}";
```

Comment:
No good idea, the mesh_code is not unique in an multi community setup. The value would be random.